resources/release-notes.md

```diff
@@ -10,6 +10,34 @@ description: |
 body="The version numbers used in headers on this page refers to the version of
 this very documentation, not to a version of any APIs described by it." %}
 
+
+## 11 Nov 2020
+
+### Version 2.0.0
+
+:tada: Launched new developer portal theme: :tada:
+*   New design :sparkles:
+*   New menu structure :orange_book:
+*   Reading time :hourglass:
+
+Other changes:
+*   Added GDPR disclaimer.
+*   Added custom 404 page.
+*   Added Transaction on File section.
+*   Added Seamless View for [Trustly][trustly-payments].
+*   Added documentation about payment states in
+    [Card Other Features][card-transaction-states]
+*   Added event overrides warning for [Card Seamless View Events][card-seamless-view]
```

Comment:
Doesn't the warning apply to all seamless view events (not just for Card)?

Reply:
Yes it does. Is it better to leave it without a link?

Reply:
No, but we can perhaps word it differently? Something like:

```suggestion
*   Added event override warning for Seamless View Events, see [Card Seamless View Events][card-seamless-view] for example.
```